gamersnet_backend/routes/posts/deletePost.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+// include our function from the database to add post
+let {deletePost} = require('../../persistence/posts');
+let {verifyUserLoggedIn} = require('../utilities/tokenUtility')
+
+let {getUserIDFromToken} = require('../../persistence/tokens.js')
+
+// this function handles the /posts/deletePost endpoint
+async function removePost(request, response) {
+    let body = request.body;
+    let cookie = request.headers.cookie;
+
+    let loggedIn = false;
+
+    if (cookie) {
+        cookie = cookie.split('=')[1];
+        loggedIn = await verifyUserLoggedIn(cookie);
+    }
+
+    if (loggedIn) {
+        let tokenDocument = await getUserIDFromToken(cookie);
+
```

Comment:
verify that the correct user(the owner) is deleting the post

Reply:
Yes, userID will be passed from here.